pkg/pillar/cmd/zedagent/handlevolume.go

```diff
@@ -59,6 +60,11 @@ func parseVolumeConfig(ctx *getconfigContext,
 		if !found {
 			log.Functionf("parseVolumeConfig: deleting %s\n", idStr)
 			unpublishVolumeConfig(ctx, idStr)
+		} else {
+			// check links from apps
+			volumeConfig := vc.(types.VolumeConfig)
+			volumeConfig.Stale = checkStaleVolume(ctx, cfgVolume, config)
```

Comment:
I think a better for volumeConfig.Stale is volumeConfig.HasNoAppReferences
(or HasAppReferences with the logic reversed) and a similar name for the check function, since there is nothing stale about creating a volume before any app is deployed to use that volume; we just currently don't do that sequence.